examples/kubernetes/configurator-values.yaml

```diff
@@ -0,0 +1,541 @@
+# -- Override the name of the chart
+nameOverride: ""
+# -- Override the full name of the release
+fullnameOverride: ""
```

Comment:
I would add merely the values needed that are changed, not everything.

Otherwise maintaining this is going to be a big burden and we risk to have two diverging files :) 

Something similar to what we do [here](https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/test/e2e/values-integration-filters.yaml)